src/vmm/src/builder.rs

```diff
@@ -309,6 +312,7 @@ pub fn build_microvm_for_boot(
         None,
         track_dirty_pages,
         vm_resources.vm_config.vcpu_count,
+        seccomp_filters.clone(),
```

Comment:
I would check the reason why `seccomp_filters` is passed here by reference. Most likely it can be passed by value, so you don't need to clone the thing.

Reply:
Yes, Patrick and I looked at this and we agree that ownership could be transferred to the VMM. It just requires changes across more of the codebase and has a minimal impact either way, so we decided to clone for the time being.

Reply:
I think the time investment here is quite small and at the same time, if this is not fixed now, it most likely will never be fixed.

Reply:
Created a low priority issue to track this: #4647